README.md

```diff
@@ -30,21 +30,47 @@ python src/car_racing.py
 
 ### Available arguments:  
 ```bash
---mode # choices=['train', 'test']
---window_length # The length of the filter window. Must be a positive integer
---memory_limit  # In bytes (?????)
+--mode # choices=['train', 'test', 'record'] # Record will make a video of the runs
+--window_length # Length of the experience replay window.
+--memory_limit  # Limit of how many observations, action, rewards and terminal states to store.
 --warmup_steps  # Lower learning rate during the warmup steps.
 --target_model_update  # Controls how often the target network is updated (n'th step). 
 --learning_rate  # Set the learning rate.
 --train_interval  # How many steps before the model re-fits the neural network
---steps  # Number of total training steps
---evaluation_episodes  # ?????
+--steps  # Number of total training steps.
+--evaluation_episodes  # Total number of test episodes.
 --load_weights_from  # Load weights from a previous run.
 ```
 
-For example, the following arguments will run the code with a limit of 10,000 steps, with 500 warmup steps during which the learning rate is lowered, and loading weights from a previous run/model. 
+**Training the Model**  
+For example, the following arguments will train the model with a limit of 10,000 steps, with 500 warmup steps during which the learning rate is lowered, and loading weights from a previous run/model. 
 ```bash
-python src/car_racing.py  --steps=10000 --warmup_steps=500 --load_weights_from=pretrained_model_v1
+python src/car_racing.py --steps=10000 --warmup_steps=500 --load_weights_from=model_1000 --mode=train
 ```
-# Any other details that you think might be useful
-Do we need to cite pieces of work we used/incorporated/found helpful?
+
+**Testing the Model**
+```bash
+python src/car_racing.py --load_weights_from=model_1000 --evaluation_episodes=10 --mode=test 
+```
+
+**Recording the Model**
+```bash
```

Comment:
I think this will break because the code iterates through sub directories. we might want to add another folder with subdirs that have weights or remove this example

Reply:
The test or record example?

Reply:
Removed the recording example. Its just for the presentation and doesn't effect how the code runs, he can still watch live runs